auto_submit/lib/service/pull_request_validation_service.dart

```diff
@@ -58,6 +58,18 @@ class PullRequestValidationService extends ValidationService {
   }) async {
     final github.RepositorySlug slug = messagePullRequest.base!.repo!.slug();
     final int prNumber = messagePullRequest.number!;
+
+    // TODO(yjbanov): figure out how to determine if the repo is MQ-enabled.
```

Comment:
more specifically: how to tell if a repo+branch is MQ-enabled; some branches will not be (e.g. release branches)

Reply:
Good point. I'll rephrase to "determine if the PR is MQ-enabled".